Formula/node.rb

```diff
@@ -107,12 +89,18 @@ def post_install
       # Dirs must exist first: https://github.com/Homebrew/legacy-homebrew/issues/35969
       mkdir_p HOMEBREW_PREFIX/"share/man/#{man}"
       rm_f Dir[HOMEBREW_PREFIX/"share/man/#{man}/{npm.,npm-,npmrc.,package.json.}*"]
-      ln_sf Dir[libexec/"npm/lib/node_modules/npm/man/#{man}/{npm,package.json}*"], HOMEBREW_PREFIX/"share/man/#{man}"
+      ln_sf Dir[libexec/"npm/man/#{man}/{npm,package.json}*"], HOMEBREW_PREFIX/"share/man/#{man}"
     end
 
     npm_root = node_modules/"npm"
     npmrc = npm_root/"npmrc"
     npmrc.atomic_write("prefix = #{HOMEBREW_PREFIX}\n")
+
+    # This is a minor hack that let's us keep a working npm
+    # in both HOMEBREW_PREFIX & also privately in libexec.
```

Comment:
Spot the `node` test builds impacting my CPU usage lol:

<img width="318" alt="screen shot 2017-06-01 at 16 25 53" src="https://cloud.githubusercontent.com/assets/6998367/26687165/05917de8-46e7-11e7-850a-063ddb488b60.png">